kube/pod.go

```diff
@@ -529,10 +528,17 @@ func getSecurityContext(role *model.InstanceGroup, createHelmChart bool) helm.No
 		// Complete the context, with conditional privileged mode
 		sc.Add("privileged", helm.NewNode(true, helm.Block(hasAll)))
 		sc.Add("capabilities", cla)
+		if allowPrivileged {
+			sc.Add("allowPrivilegeEscalation", allowPrivileged)
+		} else {
+			// We ned to allow privilege escalation if if want privileged mode
+			sc.Add("allowPrivilegeEscalation", fmt.Sprintf("{{ %s }} true {{ else }} false {{ end }}", hasAll))
```

Comment:
```suggestion
			sc.Add("allowPrivilegeEscalation", fmt.Sprintf("{{ %s -}} true {{- else -}} false {{- end }}", hasAll))
```

Don't feel strongly, but my OCD makes me suggest this change... 😄